apps/microsoft-teams/msteams-bot-service/test/mocks.ts

```diff
@@ -24,3 +24,186 @@ export const makeMockConversationBot = (): sinon.SinonStubbedInstance<Conversati
   });
   return stubbedBot;
 };
+
+export const installationUpdateAdd = {
```

Comment:
Just more of a curiosity question: do these responses look any different for a team's general channel vs. a "named" channel? I had seem some differences when I was playing around with the sample bot.